IntervalTree/IntervalTree.csproj

```diff
@@ -23,8 +24,8 @@ In computer science, an interval tree is an ordered tree data structure to hold
     <PackageReleaseNotes>BREAKING CHANGES: Removing RangeTree code after is has been renamed to IntervalTree in version 2.1.0.
 Make sure that you upgrade to version 2.1.0 first and that you have no more errors before upgrading to this version. This version contains no new features, but fixes the misleading name "RangeTree" when in fact it is an Interval tree.
 For a full list changes at https://github.com/mbuchetics/RangeTree/releases</PackageReleaseNotes>
-    <AssemblyVersion>3.0.0</AssemblyVersion>
```

Comment:
Please leave out the changes that are related to the release management. They are handled by us.

Reply:
I've reset all version numbers back to 3.0.0